downstream/modules/aap-on-azure/proc-azure-locate-aap-marketplace.adoc

```diff
@@ -6,8 +6,8 @@
 
 
 . In a browser, navigate to the Azure Marketplace.
-. Click *Private Products* from the menu on the left of the screen.
+. Click btn:[Private Products] from the menu on the left of the screen.
```

Comment:
I believe this is a menu selection and should be rewritten to:

From the navigation panel, select menu:Private Products[]